### ISSUE TITLE
Android client says it can't find pgp app when using passphrase

### ISSUE THREAD

**Issue by zoof** [keybase/keybase-issues#2979]
Linking through another attached device worked fine.

Running Cyanogenmod 13.0-20160731-NIGHTLY-d2vzw on a Samsung S3.

**Comment by DeepMac** [comment]
FYI, seen this same error on a Motorola Droid Maxx running Android 4.4.4, Kernel 3.4.42-q6ca5516
Tried installing AGP and then re-starting the passphrase login process, gives the same error message.

**Comment by zapu** [comment]
This issue should be fixed now, would you mind getting a new version and trying this again? Thank you

**Comment by DeepMac** [comment]
Yes! It works now, i forgot to update that I tried it over the weekend after I saw the app updated. Registration worked fine and chats are functional.
